content/SCALE/SCALEUIReference/VirtualizationScreens.md

```diff
@@ -57,7 +58,7 @@ The **Operating System** settings specify the VM operating system type, the time
 {{< /expand >}}
 
 ### CPU and Memory Screen
-The **CPU and Memory** settings specify the CPU mode, model, and memory size. They also let you specify the number of virtual CPUs to allocate to the virtual machine, the number of cores per virtual CPU socket, and the number of threads per core.
+The **CPU and Memory** settings specify the CPU mode, model, and memory size. They also enlist storage provisioning let you specify the number of virtual CPUs to allocate to the virtual machine, the number of cores per virtual CPU socket, and the number of threads per core.
```

Comment:
CPU and memory are about system resources/computing power or capability to process system requests and perform basic operations, it is really not about storage provisioning.
```suggestion
The **CPU and Memory** settings specify the CPU mode, model, and memory size. They also let you specify the number of virtual CPUs to allocate to the virtual machine, the number of cores per virtual CPU socket, and the number of threads per core.
```